src/test/java/guitests/IssuePanelTests.java

```diff
@@ -34,6 +34,7 @@
 public class IssuePanelTests extends UITest {
 
     private static final int EVENT_DELAY = 1000;
+    private UtilMethods util = new UtilMethods();
```

Comment:
Perhaps be more specific in the class name? e.g. `GuiTestingRobot robot` `robot.typeString("sort:date")`